assignment-1/Failed__case.md

```diff
@@ -1,87 +1,86 @@
+ 
+> At this excerpt extracted from [source] we add the question: **where could it change?**. A stricter monitoring of the deliverables by the contractors (with small and testable deliverables), could have helped this project.
+
+
+##### Cause 2: Architecture, non functional requirements.
+ 1. The system was not designed to handle the massive influx of initial users. 
+ 2. Insufficient effort made to build a system that would meet the performance and availability needs of it's stakeholders. 
+ 3. Security solutions were seemingly slapped together in a shoddy manner instead of through the kind of systematic approach that is expected in a highquality software solution.
+ 4. Many thousands of pages of legal healthcare regulations did not translated seamlessly into functional requirements.
+ 5. The basic architecture was designed and built around the notion that the system would forward requests for quotes from insurance seekers to external vendors in real-time; however, this massive interconnectivity and the subsequent burden on the government servers caused the system to collapse.
+
+>However, all these points, extracted from [1] are a compilation of the problem's triggers , but **where could it change?**
+Besides all the disastrous organizational perspective, and scoping at the missing functional requirements, the elicitation, documentation and communication of those requirements probably could have helped this project. There are many structured techniques specifically designed to learn others about the system and what should it carry, some techniques such as 'planguage' or 'volere templates'.
+
+
+ 
+##### Cause 3: Architecture, Change Managment.
+The inability to handle a growing amount of work, its potential to be enlarged in order to accommodate that growth, or a limited space to accommodate iterations can be a very dangerous thing. In this case, three months before the deadline a requirement changed.
+>**where could it change?** Finding scales and meters, RUP, and many other patterns and methods.
+
+
+##### Cause 4: Politics (Ostrich effect) - needs review. Not that great yet.
+Politicians were setting a deadline that was not realistic.  Only two weeks for the launch there were end-to-end tests. The requirement of 60,000 users was not met according to the New York Times on 30 September 2013 that based there statements on documents released by House investigators [4]. Only 11,000 users could simultaneously access the website. Also a McKinsey & Co document, made on April 4 2013, that was adressed among others to the CMS Administrator [4, page 7] stated there was insufficient time for end-to-end testing and that a “limited initial launch” would be ideal. This report only surfaced on 13 November 2013. One and a half month after the release of the website. It is unclear why this information was not used to postpone the deadline or to do a launch for a limited number of users.
```

Comment:
"Only 11,000 users could simultaneously access the website. Also a McKinsey & Co document, made on April 4 2013, that was adressed among others to the CMS Administrator [4, page 7] stated there was insufficient time for end-to-end testing and that a “limited initial launch” would be ideal." 
Very long sentence.